plugin/common/build.gradle

```diff
@@ -1,9 +1,14 @@
 apply plugin: "com.github.gmazzo.buildconfig"
 
+repositories {
+    maven { url "https://maven.mrnavastar.rocks/releases" }
+}
+
 dependencies {
     /* PROVIDED */
     compileOnly libs.multiverse
     compileOnly libs.parties.api
+    compileOnly "group.aelysium:rustyconnector-toolkit:0.8.0"
```

Comment:
The dependency should be defined in the version catalog file and just referenced here

Reply:
Resolved in commit https://github.com/ScreamingSandals/BedWars/pull/539/commits/a732f4d6e903ff6858830dd363f1f72bb9dc93bc